src/corona_hakab_model/common/agent.py

```diff
@@ -6,12 +6,12 @@
 import pandas as pd
 from numpy import nan
 
-from generation.circles import SocialCircleConstraint
+from common.social_circle import SocialCircleConstraint
```

Comment:
.social_circle instead of common.social_circle

Reply:
done